### DISCUSSION TITLE
DFplayer setup

### DISCUSSION THREAD

**Discussion by ghost** [OpenI6X/opentx#417] [Q&A] [answered]
Can anyone help me to set up the DF player?
I have connected it correctly and installed the right firmware for it. The issue I'm facing is, which audio file is to paste in the SD card. I have tried the amber audio pack but it didn't work correctly. The announcements were wrong.

**Comment by ajjjjjjjj** [comment, marked as answer]
Proper audio pack is attached to release page: https://github.com/OpenI6X/opentx/releases
instructions are here: https://github.com/OpenI6X/opentx/wiki/Modifications#dfplayer
